src/objects/deck/deck.scss

```diff
@@ -0,0 +1,52 @@
+@use '../../design-tokens/breakpoint.yml';
+@use '../../design-tokens/sizes.yml';
+@use '../../mixins/fluid';
+@use '../../mixins/media-query';
+
+/**
+ * 1. If horizontal items are shown at the wrong column count, they will appear
+ *    to break the grid. This rule keeps items densely packed so it will always
+ *    appear visually correct.
+ */
+
+.o-deck {
+  display: grid;
```

Comment:
Do we plan to implement any fallback for non-grid supporting browsers?

Right now the experience is a little odd when grid isn't supported. I'm fine if we're consciously making the choice to focus on modern browsers, but I wanted to double check this was intentional and not an oversight.

Reply:
Yes, I will add a to-do.

Note that our fallback strategy will be a bit different this time around: I don't think our goal should be to make things comparably in <= IE11, but to make the content accessible and to address low-hanging fruit.

Reply:
Totally! The main problems I had when disabling grid were:

1. Images were huge
2. no gaps between cards made it hard to tell what was connected to what

I'd guess we could pretty quickly make a fallback to address those without recreating the whole layout

Reply:
To-do added ➕